execute/plugin.go

```diff
@@ -426,17 +454,31 @@ func (p *Plugin) Outcome(
 			commitReports[i].Messages = report.Messages
 		}
 
+		outcome := exectypes.NewOutcome(state, commitReports, cciptypes.ExecutePluginReport{})
+		if outcome.IsEmpty() {
+			return nil, nil
```

Comment:
Useful to have a `Debug` log here indicating an empty outcome?

Actually should this ever happen? Should we stay in the `GetCommitReports` state until we actually have a non-empty set of commit reports to execute?

Reply:
Do you know what happens when there are errors or we form consensus on a nil outcome?

If OCR3 sets `previousOutcome` to that nil outcome then everything will work as you describe because the plugin currently treats an empty `Outcome` as `Unknown`, which transitions to `GetCommitReports`.

Reply:
It should set it to `nil` is my understanding